languages.toml

```
@@ -1460,6 +1461,16 @@ language-servers = [ "docker-langserver" ]
 name = "dockerfile"
 source = { git = "https://github.com/camdencheek/tree-sitter-dockerfile", rev = "8ee3a0f7587b2bd8c45c8cb7d28bd414604aec62" }
 
+[[language]]
+name = "docker-compose"
+scope = "source.yaml.docker-compose"
+roots = ["docker-compose.yaml", "docker-compose.yml"]
+language-servers = [ "docker-compose-langserver" ]
+file-types = ["docker-compose.yaml", "docker-compose.yml"]
```

```suggestion
file-types = [{ glob = "docker-compose.yaml"}, { glob = "docker-compose.yml" }]
```

since #8006 full filenames must use `glob`s